### PR TITLE
2021q2 Semihalf updates

### PR DESCRIPTION
**Pull request by semihalf-wojtas-marcin**

### CODE REVIEW

File: 2021q2/ena.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Hide sysctl nodes for unused IO queues
+* Add support for the large LLQ headers
+* Update HAL version
+
+Work in progress:
+
+* Introduce full kernel RSS API support
+* Allow reconfiguration of the RSS indirection table and hash key
+* Prototype the driver port to the iflib framework
+* MFC of the ENA v2.4.0 driver to the FreeBSD 11/12/13-STABLE branches
```
Comment (pauamma) [inline 2021q2/ena.adoc:27]: s/MFC of/MFC/ for grammar consistency with the other bullet items, I think.

File: 2021q2/ls1028a.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
+=== NXP LS1028A SoC support
+
+Contact: Kornel Dulęba <mindal@semihalf.com> +
+Contact: Lukasz Hajec <lha@semihalf.com> +
+Contact: Marcin Wojtas <mw@semihalf.com>
+
+The Semihalf team has been working on adding the FreeBSD support for the link:https://www.nxp.com/products/processors-and-microcontrollers/arm-processors/layerscape-processors/layerscape-1028a-applications-processor:LS1028A[NXP LS1028A] SoC.
+
+NXP LS1028A SoC is a dual-core 64-bit ARMv8 Cortex-A72 application processor with the high-speed peripherals such as 2 Time-Sensitive Networking-capable (TSN) Ethernet controllers, quad-port TSN-enabled switch, PCIE, SD/MMC, USB3.0 and others.
```
Comment (pauamma) [inline 2021q2/ls1028a.adoc:9]: s/the high-speed/high-speed/

File: 2021q2/ena.adoc (hunk 1 above)
Comment (semihalf-wojtas-marcin) [reply 2021q2/ena.adoc]: done

File: 2021q2/ls1028a.adoc (hunk 2 above)
Comment (semihalf-wojtas-marcin) [reply 2021q2/ls1028a.adoc]: done

**Comment by semihalf-wojtas-marcin** [comment]
I also added last update about the multicast routing improvements.

File: 2021q2/multicast.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+=== Multicast routing rework
+
+Contact: Wojciech Macek <wma@semihalf.com>
+
+The Semihalf team has been working on reworking existing ip_mroute module. Previous implementation was over 20 years old and didn't use modern kernel features.
+
+Complete rework of locking mechanism was done to eliminate taking multiple locks for a single job. Some portions of code were modified to use lockless constructions, BW-meter API refactored to work in a single context and routing hot path was identified and made to work efficiently.
+All these fixes helped to acheive over 5x speed boost in multicast routing.
```
Comment (pauamma) [inline 2021q2/multicast.adoc:8]: s/acheive/achieve/

Comment (semihalf-wojtas-marcin) [reply 2021q2/multicast.adoc]: done

File: 2021q2/multicast.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@ Contact: Wojciech Macek <wma@semihalf.com>
 The Semihalf team has been working on reworking existing ip_mroute module. Previous implementation was over 20 years old and didn't use modern kernel features.
 
 Complete rework of locking mechanism was done to eliminate taking multiple locks for a single job. Some portions of code were modified to use lockless constructions, BW-meter API refactored to work in a single context and routing hot path was identified and made to work efficiently.
-All these fixes helped to acheive over 5x speed boost in multicast routing.
+All these fixes helped to achieve over 5x speed boost in multicast routing.
```
Comment (pauamma) [inline 2021q2/multicast.adoc:8]: I missed this earlier (sorry): s/helped to achieve/helped achieve/

Comment (semihalf-wojtas-marcin) [reply 2021q2/multicast.adoc]: Sure, no pb - done. Thanks!

**Pull request closed without merging**